crates/type/src/tag/entry.rs

```diff
@@ -21,7 +21,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 #[serde(untagged)]
 pub enum Entry {
-    Signed(Jws),
+    Signed(Box<Jws>),
```

Comment:
Why is the box necessary here?

Reply:
I knew you would ask that. :) Clippy complains because the size difference between the `Signed` and `Unsigned` variants is too large. The other option is to disable the warning.

Reply:
I decided to disable the lint. That is also self-documenting.